lib/cli/run.js

```diff
@@ -190,6 +190,10 @@ exports.builder = yargs =>
         description: 'Run tests in parallel',
         group: GROUPS.RULES
       },
+      'posix-exit-codes': {
+        description: 'Use posix exit codes for fatal signals',
```

Comment:
[Description is wrong](description: 'Use POSIX and shell scripting exit codes as Mocha's return value',)

Reply:
https://itsfoss.com/linux-exit-codes/